babel/src/index.js

```diff
@@ -0,0 +1,3 @@
+export const add = (a, b) => {
+  return a + b;
+}
```

Comment:
Should this be named something more descriptive (e.g., "add.js") with an actual "index.js" that exports it?

Reply:
Good point.